src/common/options.cc

```diff
@@ -7645,6 +7645,10 @@ std::vector<Option> get_mds_options() {
     .set_default(64_K)
     .set_description("maximum aggregate size of extended attributes on a file"),
 
+    Option("mds_cache_trim_interval", Option::TYPE_SECS, Option::LEVEL_ADVANCED)
```

Comment:
@tchaikov any reason we can't have a TYPE_USEC? I think a smaller time granularity would be useful for some of these time based options.

Reply:
@batrick TYPE_SEC was added for better user experience. because one can use postfixes like "seconds", "hours" when specifying the options representing timespans. and in the mean while, it keeps the backward compatibility so that it's consumers can continue using it as a plain integer.

could you raise some use cases of microseconds as an option?

Reply:
I don't see an immediate need but it was something I wondered about.